java/maven/arch.xml

```diff
@@ -115,6 +115,13 @@
        will be merged into a single instance in the project's lookup.
      </p>
     </api>
+    <api group="layer" name="MavenPackagingLookup" type="export" category="devel">
+        <p>
+            <code>Projects/org-netbeans-modules-maven/&lt;packaging-type>/Lookup</code> is added to the project's additional Lookup. The content is expected
+            to contain packaing-specific services and processors, for example, <a href="@TOP@/org/netbans/modules/maven/api/execute/PrerequisitesChecker.html">PrerequisitesCheckers</a>.
+            In addition, <code>Projects/org-netbeans-modules-maven/_last/Lookup</code> defines services that act after the packaging-specific ones.
```

Comment:
Naming is hard: `_last`!? Any? Generic? Fallback? Why there is the `_`?

Reply:
Yes, it's hard. Since the name shares 'namespace' with packaging type setting in maven `pom.xml`, I wanted to include some awkward character not to clash with possible future additional packaging type. **_any** would be more descriptive, thanks.

Reply:
No further feedback, so I've changed to `_any` (154603c504) and documented the ordering in 9d7654ce75